test/fluentd-forward.sh

```diff
@@ -42,6 +42,7 @@ update_current_fluentd() {
 
     # redeploy fluentd
     os::log::debug "$( oc label node --all logging-infra-fluentd=true )"
+    os::cmd::try_until_success "oc rollout status -w ds/logging-fluentd"
```

Comment:
I don't think you can use rollout with a daemonset:
```bash
$ oc label node --overwrite --all logging-infra-fluentd-
node "ip-172-18-1-3.ec2.internal" labeled
$ oc get pods | grep fluentd
[origin@ip-172-18-1-3 artifacts]$ oc rollout status ds/logging-fluentd && echo success
daemon set "logging-fluentd" successfully rolled out
success
```
I think you can just omit this line - I think the following line is sufficient to know that fluentd is running.

Reply:
Hmmm, the master branch already has it.  What shall we do?
https://github.com/openshift/origin-aggregated-logging/blob/master/test/fluentd-forward.sh#L49

Let me double check that mux is ok since mux uses dc?
https://github.com/openshift/origin-aggregated-logging/blob/master/test/mux.sh#L318

Reply:
it should be fine to use `oc rollout` with `dc/logging-mux`.

We should fix master branch then because we cannot use `oc rollout` with `ds/logging-fluentd`

Reply:
@richm,
> We should fix master branch then because we cannot use oc rollout with ds/logging-fluentd

I've submitted one line pr for the master branch...
https://github.com/openshift/origin-aggregated-logging/pull/756